### PR TITLE
Unmanaged entity

### PR DESCRIPTION
**Pull request by hatemben**
Using ModelListType trigger error due to an unmanaged entity (No document manager defined for class Doctrine\ODM\MongoDB\PersistentCollection). Not the best way to write this.

## Subject

Using ModelListType trigger this error "No document manager defined for class Doctrine\ODM\MongoDB\PersistentCollection". So simply remove it from the list.

I am targeting this branch, because this should be fixed here.

Closes #{put_issue_number_here}

## Changelog

```markdown
### Added

### Changed

### Deprecated

### Removed

### Fixed
- Fixed unmanaged entity for ModelListType

### Security
```

### CODE REVIEW

File: src/Model/ModelManager.php [hunk 1]
```diff
@@ -279,6 +279,10 @@ public function getNormalizedIdentifier($document)
             throw new \RunTimeException('Invalid argument, object or null required');
         }
 
+        // not managed
+        if (!$this->registry->getManagerForClass(\get_class($document))) {
+            return;
+        }
```
Comment (greg0ire) [inline src/Model/ModelManager.php]: Isn't that exactly what the lines below are supposed to be about?

Comment (hatemben) [reply src/Model/ModelManager.php]: It's supposed to, but it doesn't.

Comment (greg0ire) [reply src/Model/ModelManager.php]: Maybe the condition you added should be added directly inside the condition line 287? Or maybe the issue is calling `getNormalizedIdentifier` with a collection, I don't think that it makes sense, does it?

Comment (greg0ire) [reply src/Model/ModelManager.php]: If you could provide a stack trace with the error you are getting, we might be able to figure out why this is happening.

**Pull request closed without merging**